### PR TITLE
Optionally prevent unavailable services to be registered.

### PR DESCRIPTION
**Pull request by vicsufer**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

Perform basic healthcheks prior to registering services in consul.

### Benefits
Beneficts when setting sanity check on.
- Enforce the usage of healthcheks
- Do not register not proper configured services.

### Possible Drawbacks
None

### Applicable Issues
#50

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/consul_role/tree/develop)
+### Added
+- *[#50](https://github.com/idealista/consul_role/issues/50) Add consul_service_sanity_healthcheck variable to enforce mandatory healtcheck on services* @vicsufer
```
Comment (dortegau) [inline CHANGELOG.md:8]: There is a small typo: healt**h**check

Comment (vicsufer) [reply CHANGELOG.md]: Fixed

**Review by jperera — APPROVED** [review]

File: defaults/main.yml [hunk 2]
```diff
@@ -64,6 +64,7 @@ consul_acl_agent_token: agent
 
 # Services #
 consul_agent_services: false
+consul_service_sanity_healthcheck: true
```
Comment (dortegau) [inline defaults/main.yml:67]: I think it would be nice to add a comment explaining briefly what behavior has this variable

**Review by blalop — APPROVED** [review]

File: defaults/main.yml (hunk 2 above)
Comment (vicsufer) [reply defaults/main.yml:67]: Probably explanation in readme would be better.

**Pull request merged**